src/app/app.module.ts

```diff
@@ -11,6 +11,7 @@ import { getFirestore, provideFirestore } from '@angular/fire/firestore';
 import { FIREBASE_OPTIONS } from '@angular/fire/compat';
 import { ProfileModule } from './features/profile/profile-page.module';
 import { MainModule } from './features/main/main.module';
+import { PostModule } from './features/post/post.module';
```

Comment:
Lazy loading wont work, if you import your module here. It may work without importing.